requirements/hpu.txt

```diff
@@ -13,3 +13,6 @@ habana_gpu_migration>=1.18.0
 #habana-pyhlml
 #habana_quantization_toolkit
 #habana-torch-dataloader
+
+# Extra dependencies for Intel Gaudi cards
+instructlab-training[hpu]>=0.5.0
```

Comment:
This should actually be held until 0.6.0 is cut and exists, since 0.6.0 will be the first version to feature the hpu extra

Reply:
So the version here should actually be `0.6.0` instead of `0.5.0`?

Reply:
@JamesKunstle please fix that when you get a chance, ty!

Reply:
v0.6.0 has been cut now, so can be used as min

Reply:
Got it will update asap!